apps/etterlatte-behandling/src/main/kotlin/behandling/tilbakekreving/TilbakekrevingService.kt

```diff
@@ -260,7 +260,12 @@ class TilbakekrevingService(
             throw TilbakekrevingFeilTilstandException("Tilbakekreving kan ikke attesteres fordi vedtak ikke er fattet")
         }
 
-        runBlocking { brevApiKlient.ferdigstillVedtaksbrev(tilbakekrevingId, behandling.sak.id, brukerTokenInfo) }
+        if (behandling.sendeBrev) {
+            logger.info("Sender vedtaksbrev for tilbakekreving=$tilbakekrevingId")
+            runBlocking { brevApiKlient.ferdigstillVedtaksbrev(tilbakekrevingId, behandling.sak.id, brukerTokenInfo) }
+        } else {
+            logger.info("Skal ikke sende vedtaksbrev for tilbakekreving=$tilbakekrevingId")
+        }
```

Comment:
Hmm, så det opprettes brev som potensielt kan bli ferdigstilt selv om det ikke skal sendes brev? Hvorfor? Og burde det i så fall vært slettet?

Reply:
Laget en egen oppgave for å sette et brev til SLETTET dersom man først begynner på et brev, men så går tilbake til at brev ikke skal sendes: https://jira.adeo.no/browse/EY-3843.